### PR TITLE
Add new design tokens

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #137 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

Added the requested missing design tokens

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/color/border.json [hunk 1]
```diff
@@ -71,6 +71,14 @@
           "comment": "Deprecated token",
           "reference": "n/a",
           "version": "3.12.0"
+        },
+        "inverse": {
+          "value": "{color.brand.breeze.300.value}",
+          "public": false,
```
Comment (blackfalcon) [inline src/color/border.json:77]: You don't want this to be a publically useable token?

File: src/color/border.json [hunk 2]
```diff
@@ -71,6 +71,14 @@
...
+          "onDark": true,
+          "usage": "Active border color on dark backgrounds",
+          "wcag": "n/a",
+          "deprecated": true
```
Comment (blackfalcon) [inline src/color/border.json:81]: This is deprecated?

File: src/color/ui.json [hunk 3]
```diff
@@ -111,6 +111,16 @@
           "comment": "Deprecated token",
           "reference": "n/a",
           "version": "3.12.0"
+        },
+        "inverse": {
+          "value": "{color.brand.breeze.200.value}",
+          "public": false,
```
Comment (blackfalcon) [inline src/color/ui.json:117]: False?

File: src/color/ui.json [hunk 4]
```diff
@@ -111,6 +111,16 @@
...
+          "usage": "Active state color affordance on dark backgrounds",
+          "wcag": "AAA",
+          "deprecated": true,
+          "comment": "Deprecated token",
```
Comment (blackfalcon) [inline src/color/ui.json:121]: This is deprecated?

**Review by blackfalcon — DISMISSED** [review]

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**